src/sec_edgar_bulker/__init__.py

```
@@ -0,0 +1,23 @@
+"""SEC EDGAR Bulker package."""
+
+from .core import (
+    EdgarDownloader,
+    IdxDownloader,
+    HeaderGenerator,
+    ProxyManager,
+    Config,
+    setup_logging,
+    validate_config
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)